### ISSUE TITLE
Problem with moving launchers to SD Card

### ISSUE THREAD

**Issue by mckrvss** [moraroy/NonSteamLaunchers-On-Steam-Deck#15]
Hello. I have a problem with moving launchers to the SD card. I have installed launchers using the app, the script sees their location but is ultimately unable to move them.
![2023-05-13_145810](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/assets/79174374/50969c0c-dc00-47be-9fcc-be5a5084fb68)
![2023-05-13_145838](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/assets/79174374/a59a513d-b963-4609-9cbb-65ae55e9ea77)
![2023-05-13_145852](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/assets/79174374/c360dc32-a84f-4dc9-bab8-bd9ac5272882)

**Comment by moraroy** [comment]
@mckrvss if you go into your compatdata folder are there any symlinks?

**Comment by mckrvss** [comment]
I think not. There are only game's cache folders and NonSteamLaunchers folder.

**Comment by mckrvss** [comment]
> So apparently copying the whole folder to the SD card works (of course you need to change the shortcuts in the library).

Also I had to manually transfer launchers files to my SD card just like @roydbt did. I had far too little disk space to keep launchers on my main drive.

Therefore, I have two additional questions:
1. Are there any more temporary files that I should remove from the main drive if I manually changed files location?
2. At the time when I want to uninstall the launchers it would be fine if I just delete the main folder of it? (Unfortunately I won't be able to use built-in uninstaller)
